src/App.jsx

```diff
@@ -1,10 +1,71 @@
 import './App.css'
+import pokemonlogo from './assets/International_Pokémon_logo.svg.png'
+import axios from "axios";
+import {useEffect, useState} from "react";
+import Pokemon from "./components/Pokemon.jsx";
+
+
+
 
 function App() {
 
+
+
+    const [pokemon,setPokemon] = useState([])
```

Comment:
`pokemon` zal uiteindelijk een object zijn, dus dan vind ik een default waarde van `undefined` passender
```suggestion
    const [pokemon,setPokemon] = useState();
```